moveit_py/README.md

```diff
@@ -0,0 +1,28 @@
+# MoveIt 2 Python Library
+<img src="./banner.png" width="50%">
+
+`moveit_py` is a Python library for interfacing with the core functionalities of Moveit 2.
+The goal of this library is to provide a simplified interface for Moveit 2 Python users.
+
+This Python library depends on [Pybind11](https://pybind11.readthedocs.io/en/stable/index.html) for generating Python bindings.
+The project is split into the following folders:
+
+    ├── docs                    # Sphinx documentation files
+    ├── moveit_py               # Python library stubs; Python functionalities built on top of bindings
+    ├── src/moveit_py           # Pybind11 binding code
+    ├── test                    # Unit and integration testing
+
+## Tutorials
+(1) [Using Jupyter Notebooks]()
+
+(2) [Motion Planning]()
+
+(3) [Teleoperation]()
```

Comment:
This may be hard to keep in sync as more tutorials are added or jogged around. Might be easier to just link to the entire MoveIt 2 tutorials doc and have people do their own exploring there?

Reply:
Agreed, I will update this now.

Reply:
https://github.com/ros-planning/moveit2/pull/1546/commits/eaf400e79675e1c807739366593b20217f09695e